bundles/org.openhab.binding.teleinfo/src/test/resources/cbetm-ejp-option-1.raw

```diff
@@ -1,20 +1,20 @@
-MOTDETAT 000000 B
-
-ADCO XXXXXXXXXX G
-OPTARIF EJP. "
-ISOUSC 30 9
-EJPHN 001111111 ,
-EJPHPM 002222222 B
-PTEC HN.. ^
-IINST1 010 I
-IINST2 005 N
-IINST3 008 R
-IMAX1 038 ;
-IMAX2 042 7
-IMAX3 044 :
-PMAX 17480 :
-PAPP 05800 .
-MOTDETAT 000000 B
-PPOT 00 #
-
+MOTDETAT 000000 B

```

Comment:
this one too

Reply:
Sorry don't know why.....how can I take it back?

Reply:
revert to origin